htsohm/simulation/methane_loading.py

```diff
@@ -35,19 +35,21 @@ def write_raspa_file(filename, run_id, material_id, helium_void_fraction ):
 
 def parse_energy(output_file, line_number, interaction, results):
     with open(output_file) as origin:
-        line = origin[line_number]
-        if interaction == 'host_host':
-            results['ml_host_host_avg'] = float(line.split()[1])
-            results['ml_host_host_vdw'] = float(line.split()[5])
-            results['ml_host_host_cou'] = float(line.split()[7])
-        elif interaction == 'adsorbate_adsorbate':
-            results['ml_adsorbate_adsorbate_avg'] = float(line.split()[1])
-            results['ml_adsorbate_adsorbate_vdw'] = float(line.split()[5])
-            results['ml_adsorbate_adsorbate_cou'] = float(line.split()[7])
-        elif interaction == 'host_adsorbate':
-            results['ml_host_adsorbate_avg'] = float(line.split()[1])
-            results['ml_host_adsorbate_vdw'] = float(line.split()[5])
-            results['ml_host_adsorbate_cou'] = float(line.split()[7])
+        line_counter = 1
+        for line in origin:
+            if line_counter == line_number:
```

Comment:
This parse_energy method was removed in a later commit. I have energy data for the latest runs as well!